packages/shared/babel-ast-utils/src/babelErrorUtils.js

```diff
@@ -17,8 +17,12 @@ export async function babelErrorEnhancer(
   asset: BaseAsset,
 ): Promise<BabelError> {
   if (error.loc) {
+    let start = error.message.startsWith(asset.filePath)
+      ? asset.filePath.length + 1
+      : 0;
     error.message = error.message
-      .slice(asset.filePath.length + 1, error.message.indexOf('\n'))
+      .slice(start)
+      .split('\n')[0]
       .trim();
   }
```

Comment:
Apparently Babel changed their error message format. I was getting
```
🚨 Build failed.
@parcel/transformer-js: operties' (7:3
SyntaxError: operties' (7:3
    at Parser._raise (parcel/node_modules/@babel/parser/src/parser/error.js:60:45)
    at Parser.raiseWithData (parcel/node_modules/@babel/parser/src/parser/error.js:55:17)
    at Parser.expectPlugin (parcel/node_modules/@babel/parser/src/parser/util.js:145:18)
    at Parser.parseClassProperty (parcel/node_modules/@babel/parser/src/parser/statement.js:1617:12)
    at Parser.pushClassProperty (parcel/node_modules/@babel/parser/src/parser/statement.js:1522:30)
    at Parser.parseClassMemberWithIsStatic (parcel/node_modules/@babel/parser/src/parser/statement.js:1408:14)
    at Parser.parseClassMember (parcel/node_modules/@babel/parser/src/parser/statement.js:1312:10)
    at callback (parcel/node_modules/@babel/parser/src/parser/statement.js:1236:14)
    at Parser.withTopicForbiddingContext (parcel/node_modules/@babel/parser/src/parser/expression.js:2594:14)
    at Parser.parseClassBody (parcel/node_modules/@babel/parser/src/parser/statement.js:1213:10)
_issues/5510-class-this/a.js:7:3
  6 |   }
> 7 |   y = (v) => {
>   |    ^
  8 |     return this.x(v);
  9 |   };
```

instead of `This experimental syntax requires enabling the parser plugin: 'classProperties' (7:3)`